frontend/src/components/elements/CodeBlock/CodeBlock.tsx

```diff
@@ -78,9 +80,11 @@ export default function CodeBlock({
 
   return (
     <StyledCodeBlock className="stCodeBlock">
-      <StyledCopyButtonContainer>
-        <CopyButton text={value} />
-      </StyledCopyButtonContainer>
+      {value && value && (
```

Comment:
heads up that there's a typo here duplicating the `value &&` (although I guess the second PR will fix it anyway)